packages/@lwc/babel-plugin-component/vitest.config.mjs

```diff
@@ -0,0 +1,20 @@
+import { mergeConfig, defineProject } from 'vitest/config';
+import sharedConfig from '../../../vitest.shared.mjs';
+import pkg from './package.json';
+
+export default mergeConfig(
+    sharedConfig,
+    defineProject({
+        test: {
+            name: 'babel-plugin-component',
+            // Workaround to fix `const enum`, which is required because we use e.g. `APIFeature` from `@lwc/shared`
+            // See https://github.com/vitest-dev/vitest/discussions/3964
+            // Using `src` also ensures that the test coverage is accurately reported
+            alias: Object.fromEntries(
+                Object.keys(pkg.dependencies ?? {})
+                    .filter((dep) => dep.startsWith('@lwc/'))
+                    .map((dep) => [dep, `${dep}/src`])
+            ),
```

Comment:
Does it matter if we define an alias for a package that's not actually used? Could we just move this to the root config and define aliases for all `@lwc/` packages?

Reply:
I tried doing this, but it messed with the `lwc` tests because it tries to import from `dist`.

I can give this another shot, though.

Reply:
Got it! 3b2954e55